docs/internal-documentation/distrib-testing/Debian-build-environment.md

```diff
@@ -24,7 +17,11 @@ This instruction is for creating it on Debian. See other files for other OSs.
 3. Install prerequisites
 
    ```sh
-   sudo apt-get install cpanminus gettext
+   sudo apt-get install git cpanminus gettext autoconf automake build-essential libdevel-checklib-perl libmodule-install-xsutil-perl libssl-dev libidn2-dev libtool
    ```
 
+4. Clone 'develop' branch from all Zonemaster repositories
 
+   ```sh
+   for d in ldns engine cli backend gui; do git clone -b develop https://github.com/zonemaster/zonemaster-$d.git; done
```

Comment:
This machine is for creating the Perl distribution files. I guess GUI should be excluded.

Reply:
True, it is specified down the line in https://github.com/zonemaster/zonemaster/blob/develop/docs/internal-documentation/maintenance/ReleaseProcess-create-test-distribution.md#7-produce-distribution-zip-file.
Updated.